lib/hpke/include/hpke/certificate.h

```diff
@@ -54,12 +55,15 @@ struct Certificate
   ParsedName subject() const;
   bool is_ca() const;
   ExpirationStatus expiration_status() const;
-
   std::optional<bytes> subject_key_id() const;
   std::optional<bytes> authority_key_id() const;
   std::vector<std::string> email_addresses() const;
   std::vector<std::string> dns_names() const;
   bytes hash() const;
+  std::chrono::system_clock::time_point not_before() const;
+  std::chrono::system_clock::time_point not_after() const;
+  Signature::ID public_key_algo() const;
+  Signature::ID signature_algo() const;
```

Comment:
```suggestion
  Signature::ID public_key_algorithm() const;
  Signature::ID signature_algorithm() const;
```

... and corresponding changes below.